jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -40,23 +50,24 @@
  */
 public class StepsRunner {
```

Comment:
Note to reviewers: this is the central and the only place for async scheduling and dependent input management in the new design.